### PR TITLE
Use python 2.7.13 on the worker nodes

### PR DESCRIPTION
**Pull request by amaltaro**
WMAgent version of #7807
If we decide to go with the new COMP python build, then we need to add this PR to the agent.
However, before that we need to have it deployed on CVMFS and OSG sites.

### CODE REVIEW

**Comment by ericvaandering** [comment]
Seems to me we can be smarter than having to update version numbers every time. :-) Can we auto discover this?

**Comment by amaltaro** [comment]
We can always be smarter, though it depends on the cost-benefit, which for this case is very small :-) 
Anyways, I'll update this patch such that it looks for the latest python release and just use that. That will require special hooks for the moment we migrate to python 3 as well.

**Comment by amaltaro** [comment]
Eric, the second commit has the changes you suggested. I still need to test it though.

**Comment by amaltaro** [comment]
Tested both at FNAL and CERN and it works Ok, though I still want to test it in other US and EU sites. @ticoann good for a review though.

File: etc/submit.sh [hunk 1]
```diff
@@ -1,4 +1,72 @@
...
+{
+  if [ -z "$prefix" ]; then
+    echo "ERROR: Python directory is empty"
+    exit 6
+  elif [ -z "$suffix" ]; then
+    echo "ERROR: Suffix for python directory is empty"
+    exit 7
+  fi
+
+  local latest='2.7.6'         # default release
```
Comment (ticoann) [inline etc/submit.sh]: shouldn't we set default to 2.7.13 ?

**Comment by ericvaandering** [comment]
test this please

**Comment by ericvaandering** [comment]
(testing Jenkins, not the PR)

**Comment by ericvaandering** [comment]
test this please

File: etc/submit.sh (hunk 1 above)
Comment (amaltaro) [inline etc/submit.sh]: It doesn't exist in cvmfs yet, that's why I made an existent one as default.

Comment (ticoann) [inline etc/submit.sh]: I see. Thanks,

**Comment by amaltaro** [comment]
[Fix #7789 - Use python 2.7.13 on the worker nodes](https://trello.com/c/eBjWafrE/29-fix-7789-use-python-2-7-13-on-the-worker-nodes)

**Comment by amaltaro** [comment]
And I need to make sure it can properly handle a "version" directory like `2.7.11-comp`. I'll test it tomorrow.

**Comment by amaltaro** [comment]
sigh... I rebased it with master instead of the wmagent branch!

**Comment by amaltaro** [comment]
@vkuznet can you point me to the script you are talking about in the meeting?

**Comment by vkuznet** [comment]
/cvmfs/cms.cern.ch/common/das_client

**Comment by amaltaro** [comment]
Thanks, Valentin. Much easier indeed.

**Comment by amaltaro** [comment]
Seems to be working fine. I tested it in a couple US sites, Vienna, Bari, Jinr and there were no issues.
I'm merging it now in order to cut a new wmagent tag. However, please feel free to review it.

**Pull request merged**